packages/ui-stencil/src/components/internal/orama-modal/orama-modal.tsx

```diff
@@ -137,37 +131,6 @@ export class OramaModal {
     }
   }
 
-  handleArrowNavigation(event: KeyboardEvent) {
```

Comment:
I fear we still need these to handle keyboard navigation here :( (I tried adding some links to the content, but key navigation was not working) What about moving it to utils/lib functions so we can share this across components? Or maybe also creating a base component class that includes these common accessibility behavior and then extend this class in the individual components that need it?

Reply:
I'm not sure about what you mean. The ArrowNavigation handling has been moved to the search-box component and it works for both the modal and the embed. Did you try it in [Storybook](https://orama-ui-components-storybook-git-feat-orm-1-59d909-oramasearch.vercel.app/?path=%2Fstory%2Fcomponents-searchbox--search-box)? 
Btw, we can definitely move it to a utils/lib to clean up the component's code a bit. I also like the solution about extending the component class to "share" common functionalities, but I don't think it's required in this case.

Reply:
@g-francesca I've moved this part to `@/utils/utils` for now: https://github.com/askorama/orama-ui-components/pull/45/commits/ba4f5cca9c8c5af38d98a33970ac09d8a5e5df50